test/integration/infrastructure/common.go

```diff
@@ -24,12 +24,9 @@ const (
 	natGatewayCIDR = "10.250.128.0/21" // Enhanced NatGateway need bind with VSwitch, natGatewayCIDR is used for this VSwitch
 	natGatewayType = "Enhanced"
 
-	secretName = "cloudprovider"
-
-	availableStatus = "Available"
-
-	sshPublicKey       = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDSb1DJfupnWTfKJ0fmRGgnSx8A2/pRd5oC49qE1jFX+/J9L01jUyLc5sBKZXVkfU5q5h0JfbkhJXSIkzqE+rNPnJBI4e+8Lo2TVWLAvVZRA9Fg9Dk3mgkdVB+9qW2mIqtJF5GOWKuk7HkObwpY1pX8kHC/LJVfpNQpVBqWef0WJj6vbyjhlZ3vgRxK9I6wdJzjUYtNsDhvvBTy/IBg/xp82w9T2r3GVfnaTLMeQCW9mPviDKnQsrWMgVb2A0Z4c62EbzzLzQV4ScVJ6JMgOgkMqEPdbnKF8dEQcSu+/DQZoZt56Aeov7T4oamahj9/rIDX+WR1nOcfntIdhCyoB4lISkNFz/MlPC7O8HwJk4P7rojLGNk6xmn6NxY5CJGC2dVxFsb1bmm+fKHAp62mgwEoFZcDyIkcsmnmnID9u0rJNyMz84YUGZ/jEz8LePujDHcXiqgoLsKJ8gNRneISL9+m9s1VK7WxDDIbq8iWzR7XfAVE/GzKpVYkqrWCvjKEeFIDuDUnf3jghQCQMsXnJM7zGWr1tl+Dvl2Avxmj2xyUJXYHbXbl2aM434DgQySnV8JPzYH7EsTmvuhdb8SJIbb/NonFsSM+72HpSzVc083x4B++VL7oP1X8cly62pFVM1fi8sxBio48Hq5SmAUu9T4wUY4J+AKU6osFA/ATlMCIiQ== your_email@example.com"
-	sshPublicKeyDigest = "b9b39384513d9300374c98ccc8818a8b"
+	secretName           = "cloudprovider"
+	enableEncryptedImage = false
```

Comment:
Wrt to commit 4fb0594119aea08dbe7be3113542ed5ca339626e, do we disable the encrypted image testing with this commit? If yes, what is the reason for that? In general disabled/skipped test equals no test -> no way to catch any potential issue/regression. If we have issues/flakes with this test shouldn't we follow-up in separate issue/PR?

Reply:
It takes too long to encrypt an image. There is an bug in AliCloud. Even the encrypted image is deleted, there is OSS file still remaining. With time, there will be much leftovers.

And also, the encrypted image is covered by integration test. I would suggest to disable it for now.

Reply:
> And also, the encrypted image is covered by integration test.

The test you mention here is a test executed by testmachinery against existing Gardener landscapes. I would say that both tests are different. We can execute the current test against a PR and in this way catch any potential issues on PR level. The other test is executed against a Gardener landscape and it is usually executed against released version of provider-alicloud. So the fact that there is a testmachinery test for the scenario does not allow us to drop/disable the current envtest.

> It takes too long to encrypt an image. There is an bug in AliCloud. Even the encrypted image is deleted, there is OSS file still remaining. With time, there will be much leftovers.

If there are issues with the test, then please create an issue on provider-alicloud where you properly describe the problems, the actions you took for resolution so far and action items that are needed for enabling the test again. This should be the preferred approach.

Reply:
We reported to Ali regarding this issue. OK, take your suggestion we enable it.